k8s/test-environment/deploy-test.yaml

```diff
@@ -0,0 +1,196 @@
+  ALLOWED_HOSTS: "${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}-test.geek.zone"
+  STRIPE_SECRET_KEY: "${TEST_STRIPE_SECRET_KEY}"
+  STRIPE_PUBLIC_KEY: "${TEST_STRIPE_PUBLIC_KEY}"
+  SAND_PRICE_ID: "price_1ImwyKJh8KDe9GPFrw4mWFai"
+  DONATION_PRODUCT_ID: "prod_HnmOmrwV7zDBfw"
+  GMAIL_APP_PASSWORD: "$GMAIL_APP_PASSWORD"
+  DATABASE_HOST: "postgres"
+  DATABASE_PORT: "5432"
+  DATABASE_NAME: "geekzone"
+  DATABASE_USER: "${TEST_POSTGRES_USER}"      
+  DATABASE_PASSWORD: "${TEST_POSTGRES_PASSWORD}"
+  CELERY_BROKER_URL: "${CELERY_BROKER_URL}"
+  #CELERY_BACKEND: "amqp://h3_ZfqkTGiGmQIU5U6Y5YMsC-hKz6CyL:aWP_KIM-ZIZFNmixt8GBbnigKM5oWPFb@172.20.159.27"
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-secrets 
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+type: Opaque
+stringData: 
+  POSTGRES_DB: "geekzone"
+  POSTGRES_USER: "${TEST_POSTGRES_USER}"
+  POSTGRES_PASSWORD: "${TEST_POSTGRES_PASSWORD}"
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-gz-web
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+  labels:
+    app: gz-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gz-web
+  template:
+    metadata:
+      labels:
+        app: gz-web
+    spec:
+      initContainers:
+      - name: wait-for-migration
+        image: geekzone/backend:0.1.${CIRCLE_PREVIOUS_BUILD_NUM}
+        command: ['python3', 'manage.py', 'migrate']
+        envFrom:
+          - secretRef:
+              name: testing-secrets
+      - name: wait-for-database
+        image: geekzone/backend:0.1.${CIRCLE_PREVIOUS_BUILD_NUM}
+        command: ['python3', 'manage.py', 'wait_for_database']
+        envFrom:
+          - secretRef:
+              name: testing-secrets 
+      containers:
+      - name: django-backend
+        image: geekzone/backend:0.1.${CIRCLE_PREVIOUS_BUILD_NUM}
+        envFrom:
+          - secretRef:
+              name: testing-secrets
+        resources:
+          limits:
+            memory: "100Mi"
+            cpu: 50m
+          requests:
+            memory: "50Mi"
+            cpu: 30m
+      - name: frontend-proxy
+        image: geekzone/frontend:0.1.${CIRCLE_PREVIOUS_BUILD_NUM}
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+        readinessProbe:
+          tcpSocket:
+            port: 8080
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        livenessProbe:
+          tcpSocket:
+            port: 8080
+          initialDelaySeconds: 15
+          periodSeconds: 20
+        resources:
+          limits:
+            memory: "50Mi"
+            cpu: 50m
+          requests:
+            memory: "30Mi"
+            cpu: 30m
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: test-gz-web
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}-test.geek.zone
+#    external-dns.alpha.kubernetes.io/ttl: "120" #optional
```

Comment:
Can we avoid commented code in master please?

Reply:
I'll delete this PR and create a new one